django_redshift_backend/base.py

```diff
@@ -488,27 +487,8 @@ def __init__(self, *args, **kwargs):
         self.introspection = DatabaseIntrospection(self)
         self.validation = BaseDatabaseValidation(self)
 
-    def init_connection_state(self):
```

Comment:
I think removing this method means always `SET TIME ZONE` will be send even if user does not want to.
Does any unexpected behavior happen to users who have not used the TIMEZONE so far?
(I'll read https://aws.amazon.com/jp/about-aws/whats-new/2016/09/amazon-redshift-introduces-new-data-type-to-support-time-zones-in-time-stamps/)

Reply:
I confirmed. `SET TIME ZONE` doesn't affect to `TIMESTAMP` data type.
```
> CREATE TABLE tz (wtz TIMESTAMPTZ, wotz TIMESTAMP);
CREATE TABLE

> \d tz;
                Table "public.tz"
 Column |            Type             | Modifiers 
--------+-----------------------------+-----------
 wtz    | timestamp with time zone    | 
 wotz   | timestamp without time zone | 

> INSERT INTO tz (wtz, wotz) VALUES ('2017-08-26 0:00:00', '2017-08-26 0:00:00');
INSERT 0 1

> SELECT * FROM tz;
          wtz           |        wotz         
------------------------+---------------------
 2017-08-26 00:00:00+00 | 2017-08-26 00:00:00
(1 row)

> SET TIME ZONE 'Asia/Tokyo';
SET

> INSERT INTO tz (wtz, wotz) VALUES ('2017-08-26 0:00:00', '2017-08-26 0:00:00');
INSERT 0 1

> SELECT * FROM tz;

          wtz           |        wotz         
------------------------+---------------------
 2017-08-26 09:00:00+09 | 2017-08-26 00:00:00
 2017-08-26 00:00:00+09 | 2017-08-26 00:00:00
(2 rows)
```